Week-2/ANIMATION/index.css

```diff
@@ -0,0 +1,68 @@
+.content{
+  position: absolute;
+  left:815px;
+  top:460px;
+  font-size: 1.2rem;
+  z-index: -1;
+  width:100px;
+  height:100px;
+}
+p{
+  position: fixed;
+  left:810px;
```

Comment:
please do not use such high values. it makes the UI tightly coupled and wouldn't work exactly on different browsers.
please use different positions to get the element at right position